pkg/cnidel/cniconfs.go

```diff
@@ -45,25 +45,18 @@ func readCniConfigFile(netInfo *danmtypes.DanmNet) ([]byte, error) {
 func getSriovCniConfig(netInfo *danmtypes.DanmNet, ipamOptions danmtypes.IpamConfig, ep *danmtypes.DanmEp) ([]byte, error) {
   vlanid := netInfo.Spec.Options.Vlan
   sriovConfig := sriovNet {
-    Name:   netInfo.Spec.NetworkID,
-    Type:   "sriov",
-    PfName: netInfo.Spec.Options.Device,
-    IfName: ep.Spec.Iface.Name,
-    L2Mode: true,
-    Vlan:   vlanid,
-    Dpdk:   DpdkOption{},
-    Ipam:   ipamOptions,
+    Name:      netInfo.Spec.NetworkID,
+    Type:      "sriov",
+    PfName:    netInfo.Spec.Options.Device,
```

Comment:
I think in the new version there is anyway only Master: 
https://github.com/intel/sriov-cni/blob/master/pkg/types/types.go#L22

we have agreed to integrate from commit hash. right?

Reply:
Fixed

Reply:
ohmm, I'm not sure this will be okay. IMO CNI expects the name of the physical device from the host, not the logical resource name from Kubernetes
my comment was simply that we can figure out pretty easily the name of the PF based on the DeviceID, right? If yes, let's just do that, instead of expecting the network administrator to know which host_device the Kubernetes belongs maps to. that anyway only works as long as the mapping is 1:1

Reply:
Thanks. Most probably you are right so I modified the code accordingly.
BTW: this was the original implementation

Reply:
yeah it actually works - as long as you, the developer, who know this stuff intimately- configures it.
once Average Joe starts using your feature, and as his first step misconfigures the name of the PF in DanmNet, that's when trouble starts :)

better to limit the possibilities of user misconfiguration, less fault tickets for us.
in case of NetworkType: sriov we will simply ignore Device from now on